sklearn/metrics/pairwise.py

```diff
@@ -1678,8 +1678,11 @@ def pairwise_kernels(X, Y=None, metric="linear", filter_params=False,
         If metric is "precomputed", X is assumed to be a kernel matrix.
         Alternatively, if metric is a callable function, it is called on each
         pair of instances (rows) and the resulting value recorded. The callable
-        should take two arrays from X as input and return a value indicating
-        the distance between them.
+        should take two rows from X as input and return a value indicating
+        the distance between them. This means that callables from
```

Comment:
Distance between them isn't right

Reply:
should be kernel?

Reply:
Fittingly I was just listening to this: https://www.youtube.com/watch?v=Qv5fqunQ_4I